lib/manager/buildkite/extract.ts

```diff
@@ -43,8 +43,9 @@ export function extractPackageFile(content: string): PackageFile | null {
           if (gitPluginMatch) {
             logger.debug('Examining git plugin');
             const { registry, gitPluginName } = gitPluginMatch.groups;
+            const gitDepName = gitPluginName.replace('.git', '');
```

Comment:
```suggestion
            const gitDepName = gitPluginName.replace(regEx('\\.git$'), '');
```
So we only trim the end. But I think we should update the extract regEx instead and ignore the suffix there